Corona-Warn-App/src/main/java/de/rki/coronawarnapp/main/CWASettings.kt

```diff
@@ -42,6 +42,20 @@ class CWASettings @Inject constructor(
         ).let { raw -> ConfigData.DeviceTimeState.values().single { it.key == raw } }
         set(value) = prefs.edit { putString(PKEY_DEVICE_TIME_LAST_STATE_CHANGE_STATE, value.key) }
 
+    var numberOfRemainingSharePositiveTestResultReminders: Int
+        get() = prefs.getInt(PKEY_POSITIVE_TEST_RESULT_REMINDER_COUNT, Int.MIN_VALUE)
+        set(value) = prefs.edit { putInt(PKEY_POSITIVE_TEST_RESULT_REMINDER_COUNT, value) }
+
+    val isNotificationsRiskEnabledFlow = prefs.createFlowPreference(
```

Comment:
should we use the Flow postfix? I feel like this could confuse people expecting to use this var as a flow instead of usings its .flow property

Reply:
good point, updated